workstreams/automation-architect/src/workflow-engine.ts

```
@@ -0,0 +1,569 @@
+    throw lastError;
+  }
+
+  /**
+   * Evaluate conditional expressions
+   */
+  private evaluateCondition(condition: string, context: WorkflowContext): boolean {
+    try {
+      // Simple expression evaluation
+      // In production, use a proper expression evaluator
+      const expression = this.resolveTemplate(condition, context);
+      return eval(expression);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)